internal/fields/mappings_test.go

```diff
@@ -647,7 +723,7 @@ func TestComparingMappings(t *testing.T) {
 			)
 			require.NoError(t, err)
 
-			errs := v.compareMappings("", c.preview, c.actual)
+			errs := v.compareMappings("", false, c.preview, c.actual)
```

Comment:
Added a new parameter here to know in which cases `properties` could be part of the parameters of a mapping or a subfield named `properties`.